packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -368,7 +369,12 @@ export abstract class ApplicationLoadBalancedServiceBase extends CoreConstruct {
     if (props.desiredCount !== undefined && props.desiredCount < 1) {
       throw new Error('You must specify a desiredCount greater than 0');
     }
-    this.desiredCount = props.desiredCount || 1;
+
+    if (!this.node.tryGetContext(cxapi.ECS_REMOVE_DEFAULT_DESIRED_COUNT)) {
```

Comment:
It may be helpful for posterity to add a comment on the desired behavior for v2, when we remove the feature flag.

Reply:
The updated README and the comment above the ECS_REMOVE_DEFAULT_DESIRED_COUNT feature flag both have documentation around what the desired behaviour is.

Reply:
I was referring to adding a `//TODO` in line to make it easy to pinpoint where the refactors need to happen in the future, as I don't see any harm to make it more obvious for maintenance purposes. This wasn't about the clarity of the README/docstring.